pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java

```diff
@@ -324,18 +320,12 @@ public DispatchRate getPoliciesDispatchRate(BrokerService brokerService) {
 
     public static Optional<Policies> getPolicies(BrokerService brokerService, String topicName) {
         final NamespaceName namespace = TopicName.get(topicName).getNamespaceObject();
-        final String path = path(POLICIES, namespace.toString());
-        Optional<Policies> policies = Optional.empty();
         try {
-            ConfigurationCacheService configurationCacheService = brokerService.pulsar().getConfigurationCache();
-            if (configurationCacheService != null) {
-                policies = configurationCacheService.policiesCache().getAsync(path)
-                        .get(brokerService.pulsar().getConfiguration().getZooKeeperOperationTimeoutSeconds(), SECONDS);
-            }
+            return brokerService.pulsar().getPulsarResources().getNamespaceResources().getPolicies(namespace);
         } catch (Exception e) {
             log.warn("Failed to get message-rate for {} ", topicName, e);
+            return Optional.empty();
```

Comment:
Do we want to keep catch `Exception` ?

Reply:
I know it's bad but I didn't want to get sidetracked since it's already a big PR :)